docs/source/reference/misc.rst

```diff
@@ -9,3 +9,4 @@ Miscellaneous functions
    cupy.shares_memory
    cupy.show_config
    cupy.who
+   cupy.fromDlpack
```

Comment:
Yes I will move it. Though, I was hesitant in doing this because we say "CuPy-specific functions are placed under cupyx namespace." Perhaps we should consider creating an alias for fromDlpack in cupyx and deprecating the one in cupy? (Not in this PR 😄)

Reply:
Yes I was thinking about the same thing 😃 
For now, let's add "DLPack" section in `ext.rst` to separately the list.

Reply:
Done in 6a25f87.